src/styles/main.scss

```diff
@@ -0,0 +1,163 @@
+@import 'variables';
+@import 'stars';
+@import 'header';
+
+body {
+  margin: 0;
+  padding: 0;
+}
+
+.products {
+  width: 100%;
+  height: 100%;
+  overflow: hidden;
+}
+
+.products__wrapper {
+  margin: 0 auto;
+  padding: 50px 40px;
+  display: grid;
+  grid-template-columns: $column-size;
+  grid-template-rows: auto;
+  justify-items: center;
+  justify-content: center;
+  grid-gap: 46px 48px;
+
+  @media (min-width: 1024px) {
+    grid-template-columns: repeat(4, $column-size);
+  }
+
+  @media (min-width: 768px) and (max-width: 1023px) {
+    grid-template-columns: repeat(3, $column-size);
+  }
+
+  @media (min-width: 488) and (max-width: 767) {
+    grid-template-columns: repeat(2, $column-size);
+  }
+}
+
+.card {
+  width: 198px;
+  background-color: white;
+  border: 1px solid $border-color;
+  border-radius: 5px;
+  font-family: $roboto-font;
+  color: $text-color;
+  font-weight: 400;
+  transition: all 300ms linear;
+
+  &:hover {
+    transform: scale(1.2);
+    transition: all 300ms ease;
+  }
```

Comment:
You have the same transition property declared in multiple places. You can create a mixin to reuse the transition property and make your code DRY (Don't Repeat Yourself).